src/main/resources/modules/cystic_fibrosis.json

```diff
@@ -0,0 +1,1538 @@
+{
+  "name": "Cystic Fibrosis",
+  "remarks": [
+    "Cystic Fibrosis, or mucoviscidosis, is an autosomal recessive genetic disorder which is named after that scarring (fibrosis) and cyst formation in the pancreas.  It is characterized by thick, viscous secretions such as thickening of mucus which causes many respiratory problems.  Statistics and information were gathered from the pages and information available at:\nhttps://www.cff.org\n\nSome data and statistic came from:\nhttps://www.sciencedirect.com/science/article/pii/S1569199315002106?viewFullText=true#f0015\n\n\n\n\n\n"
```

Comment:
Great to have remarks with sources at the top level, but it would be even more useful if the individual states where there are distributed transitions had specific links to what the cited info is and where it comes from